hnn_core/parallel_backends.py

```diff
@@ -112,6 +112,10 @@ def run_subprocess(command, obj, timeout, *args, **kwargs):
         The data returned by the child process.
     """
     proc_data_bytes = b''
+    # each loop while waiting will involve two Queue.get() timeouts, each
+    # 0.01s. This caclulation will error on the side of a longer timeout
+    # than is specified because more is done each loop that just Queue.get()
+    timeout_cycles = timeout / 0.02
```

Comment:
I guess we can do more accurate `timeout` with `time.time` but we can leave this for a future PR or someone else